porting/first-steps.rst

```diff
@@ -22,9 +22,9 @@ Pick an Android target device
 
 If you're here, you probably already have a device in mind that you wish to port to. However, we still encourage you to port to devices that meet the following requirements:
 
-* **Kernel:** Halium currently requires a device with a kernel greater than or equal to version 3.1.0 - older device kernels are not compatible with the glibc build in the root filesystem being used. Check your device's kernel version by finding "Kernel Version" in the About page of your Android settings.
+* **Kernel:** Halium currently requires a device with a Linux kernel version greater than or equal to 3.10 - older device kernels are not compatible with systemd v217 or newer, `according the the systemd 217 README <https://github.com/systemd/systemd/blob/v217/README#L40>`_. Some Halium distributions may use a kernel as old as 3.4, such as Ubuntu Touch. You may find your device's kernel version by finding "Kernel Version" in the About page of your Android settings.
 * **RAM:** While 1GB of RAM is sufficient to start the OS, it is recommended to have greater than 2GB to have a good end user experience.
-* **Chipset:** Try to avoid Mediatek chipsets, they are not open-sourced and so there is rarely a usable Android source tree available for them.
+* **Chipset:** Try to avoid Mediatek chipsets, they generally do not have an Android source tree available. Kernel source availability is not enough for a Halium port.
```

Comment:
Maybe make a note if they are supported by lineageOS it's probably not an issue. I would summon @NotKit since he knows better about Mediatek.

Reply:
The biggest issue with MediaTek is not device tree, but kernel source code. I would change chipset section altogether. Both Qualcomm and MediaTek are known to work now, Exynos and HiSilicon are less explored. As additional requirement it should be listed that device should have kernel source code published by manufacturer and either official LineageOS support or device tree that can be used to build unofficial LineageOS.